Libraries/libcar/Sources/Rendition.cpp

```diff
@@ -377,14 +377,9 @@ Decode(struct car_rendition_value *value)
             return ext::nullopt;
         } else if (header1->compression == car_rendition_data_compression_magic_lzvn || header1->compression == car_rendition_data_compression_magic_jpeg_lzfse) {
 #if HAVE_LIBCOMPRESSION
-            compression_algorithm algorithm;
-            if (header1->compression == car_rendition_data_compression_magic_lzvn) {
-                algorithm = (compression_algorithm)_COMPRESSION_LZVN;
-            } else if (header1->compression == car_rendition_data_compression_magic_jpeg_lzfse) {
-                algorithm = COMPRESSION_LZFSE;
-            } else {
-                assert(false);
-            }
+            compression_algorithm algorithm = header1->compression == car_rendition_data_compression_magic_lzvn ?
+                (compression_algorithm)_COMPRESSION_LZVN :
+                COMPRESSION_LZFSE;
```

Comment:
Instead of defaulting to `COMPRESSION_LZFSE`, shouldn't we return an error or something? It seems like with your code we're gonna use LZFSE for everything other than LZVM and LZFSE, which might be a bug.